utils/mariadb/Makefile

```diff
@@ -0,0 +1,215 @@
+#
+# Copyright (C) 2017 Lucian Cristian
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/uclibc++.mk
+
+PKG_NAME:=mariadb
+PKG_VERSION:=5.5.54
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-galera-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=\
+	https://downloads.mariadb.org/interstitial/$(PKG_NAME)-galera-$(PKG_VERSION)/source \
```

Comment:
Those URLs are a bit quirky and not the official MariaDB release, please use these ones instead. (Taken off the main site).

https://mirror.netcologne.de/$(PKG_NAME)/$(PKG_NAME)-$(PKG_VERSION)/source \
https://mirror.lstn.net/$(PKG_NAME)/$(PKG_NAME)-$(PKG_VERSION)/source \
http://lon1.mirrors.digitalocean.com/$(PKG_NAME)/$(PKG_NAME)-$(PKG_VERSION)/source \
http://ftp.kaist.ac.kr/$(PKG_NAME)/$(PKG_NAME)-$(PKG_VERSION)/source \
http://ftp.utexas.edu/$(PKG_NAME)/$(PKG_NAME)-$(PKG_VERSION)/source \